### ISSUE TITLE
Updates to inputs and "You have unsaved changes"

### ISSUE THREAD

**Issue by substrae** [GCTC-NTGC/gc-digital-talent#4831]
# 🌩️ Summary

The changes to fields when it comes to "unsaved changes" are a bit extreme. The main concerns are:
- Alert block (at the top) appears immediately as someone starts typing, this moves the entire interface down immediately which is a very jarring interaction.
- Empty fields are marked as "unsaved" immediately after tabbing in and out as opposed strictly when a change has happened
- Unsaved changes label has made a weird run-on sentence
- Background and outline is too jarring color change.

# 🧪 Proposed changes
- [x] https://github.com/GCTC-NTGC/gc-digital-talent/issues/4980
- [ ] https://github.com/GCTC-NTGC/gc-digital-talent/issues/4981
- [x] #5027

![image](https://user-images.githubusercontent.com/16977254/205081860-12003429-08f3-43e3-b3c7-a0e5d6a8d884.png)

# 📑 Related issues
- #4598 
- #4832

**Comment by substrae** [comment]
@gobyrne Tagging you in case you want to add anything.

**Comment by MaddyDaigle** [comment]
@substrae Could we also pop the "unsaved changes" label in a bracket like the required fields one? I think it just looks a bit cleaner/is a bit more noticeable. As an quick chrome devtools example:

![Capture](https://user-images.githubusercontent.com/1794660/202827272-0ac3632e-9183-49e2-8083-395bdf0fea61.PNG)

**Comment by esizer** [comment]
> Could we also pop the "unsaved changes" label in a bracket like the required fields one and add the alert blue colour?

I would be cautious with the use of parentheses since JAWS reads them out to users. The example provided would be read:

"Bilingual evaluation left paren required right paren left paren unsaved changes right paren".

This may be a little too noisy for some users without providing much value.

**Comment by MaddyDaigle** [comment]
Hm.. fair - though I don't like how it's just chilling there visually.. kind of hard to parse.

**Comment by gobyrne** [comment]
I think this is related: #4809   "Error message displays before form submission"

**Comment by gobyrne** [comment]
Just one issue remaining, closing the epic
